gitpod-io/gitpod
# Add proper metrics for IDE startup (code browser)

## Description

Add proper metrics for IDE startup (code browser)

See also [[internal chat]](https://gitpod.slack.com/archives/C01KGM9BH54/p1668757178295269?thread_ts=1668748433.738219&cid=C01KGM9BH54)

## Related Issue(s)

Fixes #

## How to test

- Start workspace in preview env
- Check if ide-metrics report `gitpod_ide_starthelper_phase_duration`

**How to check ide-metrics**
- Open this PR with Gitpod
```shell
# get ide metrics pod name
kubectl get pods | grep ide-metrics

# port forward
kubectl port-forward pods/<pod_name> 9500:9500
```
- Open 9500 in the browser

## Release Notes

```release-note
NONE
```

## Documentation


## Werft options:

      If enabled this will build `install/preview`
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
